src/Sarif.FunctionalTests/SarifConverterTests.cs

```diff
@@ -103,6 +130,25 @@ private static string FormatFailureReason(StringBuilder sb, string toolName)
             return sb.ToString();
         }
 
+        private bool CanGeneratedSarifBeDeserializedToExpectedSarif(string generatedSarif, string expectedSarif)
```

Comment:
A couple of comments about this name:

1. This function is a predicate, a statement of something we expect to be true. The thing we expect to be true is that "generated SARIF can be deserialized to expected SARIF", so the function name should be `GeneratedSarifCanBeDeserializedToExpectedSarif`.

Note that this makes the code read smoothly in English:
```
if (GeneratedSarifCanBeDeserializedToExpectedSarif(actualSarif, expectedSarif)
{
    ...
```

Reply:
This method can be `static`.

Reply:
As a rule I like to put the higher-level functions first, followed by the functions that they use. So IMO we should move this function down below `RunConverter`.

Reply:
Fixed.